rfd/0062-tctl-sso-configure-command.md

```diff
@@ -0,0 +1,205 @@
+---
+authors: Krzysztof Skrzętnicki <krzysztof.skrzetnicki@goteleport.com>
+state: draft
+---
+
+# RFD 62 - `tctl sso configure` command
```

Comment:
Sorry but I stole number 62 in https://github.com/gravitational/teleport/blob/master/rfd/0062-tsh-proxy-template.md so don't forget to update this :)